config/migrations/20220926091614-beleidsdomeinen-themis-concepts.ttl

```diff
@@ -0,0 +1,124 @@
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix tl: <http://mu.semte.ch/vocabularies/typed-literals/> .
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+
+<http://themis.vlaanderen.be/id/beleidsdomein/c23a60f3-ffaf-44ef-be18-ff1792117caa> a ext:Beleidsdomein, skos:Concept ;
```

Comment:
Does `ext:Beleidsdomein` have an added value if we're going to use the generic `skos:Concept` class to fetch the data in the frontend?

Reply:
I don't think so, at least it seems we don't use it anywhere in Kaleidos related services (that I could find). Only in the [authorization config of `app-persberichten`](https://github.com/kanselarij-vlaanderen/app-persberichten/blob/master/config/authorization/config.ex#L71-L72) but I don't actually know how that relates to Kaleidos Beleids{velden,domeinen}.

Reply:
Ok, I will add a migration to remove those types before merging :+1: